src/olympia/amo/utils.py

```diff
@@ -624,7 +624,7 @@ def pngcrush_image(src, **kw):
     return False
 
 
-def resize_image(source, destination, size=None):
+def resize_image(source, destination, size=None, hero_image=False):
```

Comment:
how about the kwarg is just the format, so `format=png` and if it's not `'png'` we just pass it directly to the save, like `im.save(fp, format)`?  That makes it a bit more general in case we decide to use this for other stuff beyond the hero shelves.